src/sentry/search/eap/columns.py

```diff
@@ -34,9 +35,11 @@ class ResolvedColumn:
     # Validator to check if the value in a query is correct
     validator: Callable[[Any], bool] | None = None
 
-    def process_column(row: Any) -> None:
-        """Pull the column from row, then process it and mutate it"""
-        raise NotImplementedError()
+    def process_column(self, value: Any) -> Any:
```

Comment:
The original intent was this would mutate on the entire row returned by snuba, but since the rpc returns an array per column, its actually easier to just run the processor on the individual item instead